### ISSUE TITLE
Include BP for virtual events

### ISSUE THREAD

**Issue by ljhwang** [geodynamics/best_practices#19]
The thought is to create a list of best practices for virtual events.  Could be a separate document or part of tutorials.

**Comment by baagaard-usgs** [comment]
There are some common themes across virtual events, but workshops, user forums, and hackathons are quite different and probably need more specifics.

**Comment by gassmoeller** [comment]
I agree they are different, but I think there are enough similarities to put guidance into one list. We all had hackathons, user meetings, and workshops over the past year, and some commonalities I see were:
- Scheduling a full day does not work in virtual settings (because of time zones and other commitments), limits to 4 or 6 hours work better
- It is possible (though not encouraged) to have weekend in-person events, but participants do not support weekend events for virtual meetings
- Sessions can be at most 90 min (better 60), because it is harder to concentrate virtually
- It helps to have one common place of communication (a Google Doc, a Slack channel, whatever else) besides the Video channel to collect documentation
- Asking participants to switch on their video helps with engagement :-)

I am sure there is more, this is just what I noticed during our user meeting earlier this week.

**Comment by ljhwang** [comment]
This came up as I was recently asked about logistics for a virtual meeting (I also note there are several online resources as well) and I can up with an overlapping list:

- One person (minimum) dedicated to troubleshooting technical problems in the background
- Two moderators for every session to monitor chat and help with the ‘foreground’
- Keeping each day to 4hrs’ish. We weren’t very good at this but 8 hrs is unrealistic.
- 2 breaks of 30 mins.  15 mins turns out to be waaaay to short.
- Be adaptable about timing (but don’t sacrifice breaks). This is even a must for in person workshops!
- Give people plenty of space to interact in small groups. Zoom has recently made this easier my allowing people to roam between breakout groups.  This was not possible before so it lead to a system of creating many links so people could bounce around and recombined at will. There are other platforms that allow this but it is best to keep people in the same environment.

**Comment by ljhwang** [comment]
I also want to point to this (there are similar resources on the web) for zoom meetings: https://geodynamics.org/cig/events/calendar/2020-tectonics-workshop/zoom-meeting-tips/
